internal/commands/config.go

```diff
@@ -124,11 +130,14 @@ func (cp ConfigFactory) internalConfig(app *model.App, vmConfig vm.Config, clp c
 }
 
 // Config returns the command configuration.
-func (cp ConfigFactory) Config(app *model.App, vmConfig vm.Config, remoteConfig *remote.Config) (*Config, error) {
+func (cp ConfigFactory) Config(app *model.App, vmConfig vm.Config, remoteConfig *remote.Config, forceContext string) (*Config, error) {
```

Comment:
why not just pass `forceOptions` here

Reply:
I would have make export `forceOptions`. I thought we didn't want to expose members under `internal` outside.